### ISSUE TITLE
Surface errors when bundling tests

### ISSUE THREAD

**Issue by cowboyd** [thefrontside/bigtest#363]
Parcel output is not surfaced whenever there is an error bundling the tests into the manifest. For example, the following error was being swallowed: 

```
Server running at http://localhost:1234
⚠️  Could not load source file "../src/index.ts" in source map of "../../../../node_modules/@bigtest/interactor/dist/index.js".
⚠️  Could not load source file "../src/converge.ts" in source map of "../../../../node_modules/@bigtest/interactor/dist/converge.js".
🚨  /Users/cowboyd/Code/@frontside/release/node_modules/@bigtest/interactor/dist/converge.js:4:88: Cannot resolve dependency 'perf_hooks'
  2 | Object.defineProperty(exports, "__esModule", { value: true });
  3 | const win = (typeof (window) === 'object') ? window : {};
> 4 | const performance = (typeof (win.performance) === 'object') ? win.performance : require('perf_hooks').performance;
    |                                                                                        ^
  5 | function wait(ms) {
  6 |     return new Promise((resolve) => setTimeout(resolve, ms));
  7 | }
```

The orchestrator just never receives the next build event.

At the very minimum, we need to print out the parcel error to the console.  But that's not enough to completely solve the problem since we want to also be able to observe and react to manifest events inside stateful UI's like a web interface, or an advanced CLI console. That means showing when the bundle is in error, we need to show it in the web page, and also display along side what the content of the error is.

In other words, the bundle state should be part of the atom, and if it is currently errored out, then that should be reflected in that state.

**Comment by cowboyd** [comment]
This was fixed with the work @pittst3r did in #389
